pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/mercury/GetMercuryOrdersRoute.java

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.dsm.route.mercury;
+
+import java.util.ArrayList;
+
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.dsm.db.dao.mercury.ClinicalOrderDao;
+import org.broadinstitute.dsm.db.dao.mercury.MercurySampleDao;
+import org.broadinstitute.dsm.db.dto.mercury.ClinicalOrderDto;
+import org.broadinstitute.dsm.db.dto.mercury.ClinicalOrderUseCase;
+import org.broadinstitute.dsm.security.RequestHandler;
+import org.broadinstitute.dsm.statics.RoutePath;
+import org.broadinstitute.dsm.statics.UserErrorMessages;
+import org.broadinstitute.dsm.util.UserUtil;
+import spark.QueryParamsMap;
+import spark.Request;
+import spark.Response;
+
+@Slf4j
+public class GetMercuryOrdersRoute extends RequestHandler {
+    private MercurySampleDao mercurySampleDao;
+    public static String projectId;
+    public static String topicId;
+    private ClinicalOrderUseCase clinicalOrderUseCase = new ClinicalOrderUseCase();
+    private ClinicalOrderDao clinicalOrderDao;
+
+    public GetMercuryOrdersRoute(MercurySampleDao mercurySampleDao, ClinicalOrderDao clinicalOrderDao, String projectId, String topicId) {
+        this.mercurySampleDao = mercurySampleDao;
+        this.clinicalOrderDao = clinicalOrderDao;
+        this.projectId = projectId;
+        this.topicId = topicId;
+    }
+
+    @Override
+    protected Object processRequest(Request request, Response response, String userId) throws Exception {
+        QueryParamsMap queryParams = request.queryMap();
+        if (queryParams.value(RoutePath.REALM) == null) {
+            throw new RuntimeException("No realm query param was sent");
+        }
+        String realm = queryParams.get(RoutePath.REALM).value();
+        String userIdRequest = UserUtil.getUserId(request);
+        if (!UserUtil.checkUserAccess(realm, userId, "kit_sequencing_order", userIdRequest)) {
+            log.warn("User doesn't have access");
+            response.status(500);
+            return UserErrorMessages.NO_RIGHTS;
+        }
+        if (request.url().contains(RoutePath.GET_MERCURY_ORDERS_ROUTE)) {
```

Comment:
this seems like an unusual `if` -- shouldn't it be DSMServer/the router's responsibility to check that?  How else would this method get invoked?